physics/equipotential_test.cpp

```diff
@@ -349,36 +417,108 @@ TEST_F(EquipotentialTest, BodyCentredBodyDirection_GlobalOptimization) {
   auto const plane =
       Plane<World>::OrthogonalTo(Vector<double, World>({0, 0, 1}));
 
-  std::vector<std::vector<std::vector<Position<World>>>> all_positions;
+  std::vector<std::vector<std::vector<std::vector<Position<World>>>>>
+      all_positions;
   std::vector<std::vector<std::vector<double>>> all_βs;
+  std::vector<std::vector<Position<World>>> trajectory_positions(
+      trajectories.size());
+  std::vector<SpecificEnergy> energies;
   for (int j = 0; j < number_of_days; ++j) {
     LOG(ERROR) << "Day #" << j;
     t = t0_ + j * Day;
     CHECK_OK(ephemeris_->Prolong(t));
     all_positions.emplace_back();
     all_βs.emplace_back();
 
-    auto const maxima = optimizer.FindGlobalMaxima(
-        /*points_per_round=*/100,
+    auto const arg_maxima = optimizer.FindGlobalMaxima(
```

Comment:
`arg_` is not a great name.  `location_maxima`, maybe?

Reply:
arg_maximorum.